examples/offline/utils.py

```diff
@@ -1,5 +1,6 @@
 import d4rl
-import gymnasium as gym
+import d4rl.gym_mujoco
+import gym
```

Comment:
That's a problem, we don't use gym anymore. I'm surprised it didn't show up in tests, it's unclear why this change was necessary at all.

@BFAnas could you pls fix this? Since you're already pushing then, pls also fix the comment above regarding the docstring

Reply:
this is example, which isn't covered by test